reference/service/webhooks.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Pulumi Webhooks
+---
+
+Webhooks allow you to build applications that will be notified when events
+happen on Pulumi. For example, whenever a stack is updated, an team is modified,
+and so on. Whenever an event occurres will send an HTTP `POST` request to all
+registered webhooks. The webhook then can then be used to emit some
+notification, start running integration tests, or even update additional stacks.
+
+Webhooks can be used for pretty much anything you want, and are the foundation
+of most _ChatOps_ workflows.
+
+> Pulumi Webhooks are available for the Team and Enterprise editions of Pulumi.
+> If you’re keen to try them out, start a trial of Team Edition
+> [here](https://app.pulumi.com/site/organizations/add).
+
+<style>
+/* Custom CSS for tables; cloned from roles-and-access-controls.md */
+td, th {
+    padding: 8px 8px;
+    border: 1px solid rgba(0,0,0,0.13);
+}
+
+thead tr th {
+    color: #00acf2;  /* $primary2, blue */
+    font-weight: 800;
+}
+
+tbody tr td:first-child {
+    text-align: left;
+    width: 250px;
+}
+
+tbody tr td {
+    width: 650px;
+}
+</style>
+
+## Management
+
+Webhooks can be attached to either a stack or an organization. Stack webhooks
+will be notified whenever the stack is updated or changed. Organization
+webhooks will be notified for events happening within each of the organization’s
+stacks, as well as when organization-specific events like team or membership
+changes.
+
+> There are some restrictions for the number of webhooks that can be registered
+> when using the Pulumi Team editions. [Contact us](https://www.pulumi.com/about/#contact-us)
+> if you need the limit increased.
+
+The webhooks management page is on the stack or organization SETTINGS tab.
+
+![Organization webhooks](../../images/reference/service/webhooks/org-webhooks.png)
+
+### Creation
+
+Creating a new webhook is simply a matter of providing a _display name_, _webhook
+URL_, and an optional _shared secret_.
+
+![Stack webhooks](../../images/reference/service/webhooks/stack-webhooks.png)
+
+If a secret is provided, then webhook deliveries will be contain a signature in
```

Comment:
Are these secrets encrypted? It may be worth specifying so people know how secret their secret is. And if it's not encrypted, we may want to name it something else, to avoid confusion with the secrets management part of our product.